pymongo/bulk.py

```diff
@@ -254,25 +254,38 @@ def gen_unordered(self):
                 yield run
 
     def _execute_command(self, generator, write_concern, session,
-                         sock_info, op_id, retryable, full_result):
+                         sock_info, op_id, retryable, full_result, final_write_concern=None):
```

Comment:
Let's move final_write_concern on a new line to fit in 80chars.

Reply:
Done